core/common/src/test/scala/net/liftweb/common/BoxSpec.scala

```diff
@@ -141,6 +141,15 @@ class BoxSpec extends Specification with ScalaCheck with BoxGenerator {
     "define a 'flatMap' method transforming its value in another Box. If the value is transformed in an Empty can, the total result is an Empty can" in {
       Full(0) flatMap { x: Int => if (x > 0) Full("full") else Empty } must beEmpty
     }
+    "define a 'flatten' method if it contains another Box. If the inner box is a Full can, the final result is a Full can" in {
```

Comment:
(The word “can” is from when `Box` was named `Can`, a long time ago; haven't had a chance to clean that term out of the whole code base yet.)